### DISCUSSION TITLE
Angular Compatibility

### DISCUSSION THREAD

**Discussion by glopezfrutos** [twbs/bootstrap#37347] [Q&A] [answered]
Hi! I'd like to know if there's compatibility with Angular. Running "ng serve" I have no troubles, but when I run “ng build” the issues appear. This is the error I get:
![image 1](https://user-images.githubusercontent.com/94074380/196957705-ae397d31-2641-484c-9069-50027e2655c8.png)
In particular,  the carouse behavior changes when the project is built.
I have been searching for “legend+* -> Cannot read property 'type' of undefined” and I found that there’s an issue between Angular and something called critters: https://github.com/GoogleChromeLabs/critters/issues/103 .  I don't know how it is related with Bootstrap.

**Comment by julien-deramond** [comment, marked as answer]
Hi @glopezfrutos. More information in https://github.com/twbs/bootstrap/issues/36136 and https://github.com/twbs/bootstrap/issues/37116. It is indeed not related to Bootstrap directly and comes from https://github.com/GoogleChromeLabs/critters/issues/103 as you found out.
